ref/Microsoft.Build/net/Microsoft.Build.cs

```diff
@@ -981,6 +981,18 @@ public partial class BuildRequestData
+        public BuildRequestData(Microsoft.Build.Graph.ProjectGraph projectGraph, string[] targetsToBuild) { }
+        public BuildRequestData(Microsoft.Build.Graph.ProjectGraph projectGraph, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices) { }
+        public BuildRequestData(Microsoft.Build.Graph.ProjectGraph projectGraph, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices, Microsoft.Build.Execution.BuildRequestDataFlags flags) { }
+        public BuildRequestData(Microsoft.Build.Graph.ProjectGraph projectGraph, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices, Microsoft.Build.Execution.BuildRequestDataFlags flags, System.Collections.Generic.IEnumerable<string> propertiesToTransfer) { }
+        public BuildRequestData(Microsoft.Build.Graph.ProjectGraphEntryPoint projectGraphEntryPoint, string[] targetsToBuild) { }
+        public BuildRequestData(Microsoft.Build.Graph.ProjectGraphEntryPoint projectGraphEntryPoint, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices) { }
+        public BuildRequestData(Microsoft.Build.Graph.ProjectGraphEntryPoint projectGraphEntryPoint, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices, Microsoft.Build.Execution.BuildRequestDataFlags flags) { }
+        public BuildRequestData(Microsoft.Build.Graph.ProjectGraphEntryPoint projectGraphEntryPoint, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices, Microsoft.Build.Execution.BuildRequestDataFlags flags, System.Collections.Generic.IEnumerable<string> propertiesToTransfer) { }
+        public BuildRequestData(System.Collections.Generic.IEnumerable<Microsoft.Build.Graph.ProjectGraphEntryPoint> projectGraphEntryPoints, string[] targetsToBuild) { }
+        public BuildRequestData(System.Collections.Generic.IEnumerable<Microsoft.Build.Graph.ProjectGraphEntryPoint> projectGraphEntryPoints, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices) { }
+        public BuildRequestData(System.Collections.Generic.IEnumerable<Microsoft.Build.Graph.ProjectGraphEntryPoint> projectGraphEntryPoints, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices, Microsoft.Build.Execution.BuildRequestDataFlags flags) { }
+        public BuildRequestData(System.Collections.Generic.IEnumerable<Microsoft.Build.Graph.ProjectGraphEntryPoint> projectGraphEntryPoints, string[] targetsToBuild, Microsoft.Build.Execution.HostServices hostServices, Microsoft.Build.Execution.BuildRequestDataFlags flags, System.Collections.Generic.IEnumerable<string> propertiesToTransfer) { }
```

Comment:
On the one hand, I hate exposing these thousand overloads. On the other, I can't be bothered to figure out which ones are most important. So 👍 I guess.